monai/networks/nets/dynunet.py

```diff
@@ -183,12 +183,14 @@ def check_deep_supr_num(self):
 
     def forward(self, x):
         out = self.skip_layers(x)
-        out = self.output_block(out)
+        return self.output_block(out)
 
-        if self.training and self.deep_supervision:
-            return [out] + self.heads[1 : self.deep_supr_num + 1]
+    def get_feature_maps(self):
+        """
+        Return the feature maps if set `deep_supervision=True`.
 
-        return [out]
+        """
+        return self.heads[1 : self.deep_supr_num + 1] if self.deep_supervision else None
```

Comment:
since it's a separate method, perhaps line 193 could be `return self.heads[1 : self.deep_supr_num + 1]`. and this class doesn't need `self.deep_supervision` member.  the class docstring should be revised

Reply:
Hi @wyli ,

I updated the PR according to your comments.
Could you please help review it again?

Thanks.